config/environments/development.rb

```diff
@@ -45,6 +45,12 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  Rails.application.config.assets.precompile += [
+    'teaspoon.css',
+    'teaspoon-jasmine.js',
+    'jasmine/2.3.4.js'
+  ]
+
```

Comment:
Rails.application is niet nodig.

Waarom  staat dit trouwens in development? Moet je niet rake assets:precompile aanroepen voordat dit uberhaupt iets doet? En anders doet ie toch gewoon een on the fly compile in development? Is dit dan puur om snelheid te winnen?

Reply:
volgens mij is dit nog een artefact van onze zoektocht om alles werkend te krijgen met de laatste react. Ik zal er nog even naar kijken